static/js/components/GcnEventPage.jsx

```diff
@@ -70,11 +70,14 @@ const useStyles = makeStyles((theme) => ({
   columnItem: {
     marginBottom: theme.spacing(2),
   },
-  source: {
+  noSource: {
```

Comment:
What's with this name?!

Reply:
This is the class for an element that is displayed only when there are no sources to tell the user, seemed more logic that its called noSource then source no ?

Reply:
the sorting by `modified` for the localization is already done @mcoughlin

Reply:
Ah yeah that makes sense!

Reply:
Perhaps `noSources` make it a tiny bit more obvious?